data/strings/strings.txt

```diff
@@ -21084,7 +21086,7 @@
 
   [unable_to_calc_alert_subtitle]
     tags = android,ios
-    en = Unfortunately, we couldn't find a route, probably due to the options you have chosen. Please change the settings and try again.
+    en = Unfortunately a route cannot be found, likely due to the routing options you have enabled. Please disable some and try again.
```

Comment:
Change is better than disable. Enabling may also help. Did you test this dialog?

Reply:
changed to 'Unfortunately a route cannot be found, likely due to the routing options you have selected. Please change them and try again.'
it's about the same length as the old string, so should be ok. i can't find a route to test it with annoyingly